### ISSUE TITLE
does this tool support the dump.cs file generated by Il2CppDumper?

### ISSUE THREAD

**Issue by MartinKayJr** [QM4RS/Il2CppAnalyzer-IDA#1]
after I loaded the script using 7.7.220118 and selected the dump.cs file, there was no effect

**Comment by QM4RS** [comment]
No, this project is specifically designed to work with the `dump.cs` file extracted from `frida-il2cpp-bridge`. It does not support `il2cppdumper` or other formats.
